code/modules/mob/living/living_defense.dm

```diff
@@ -201,6 +187,24 @@
 	adjust_fire_stacks(3)
 	IgniteMob()
 
+//Share fire evenly between the two mobs
+//Called in MobBump() and Crossed()
+/mob/living/proc/spreadFire(mob/living/L)

```

Comment:
Does this mean I can hug people into the fiery inferno that I might be on while hugging? Because that's how that proc sounds and the code seems to agree.
I do have to wonder how it logically spreads fire_stacks but what's not to like here?